packages/react/src/AnchorNav/AnchorNav.tsx

```diff
@@ -214,16 +214,16 @@ function _AnchorNav({children, enableDefaultBgColor = false, hideUntilSticky = f
           <button
             onClick={handleMenuToggle}
             className={clsx(styles['AnchorNav-menu-button'])}
-            aria-expanded={menuOpen ? 'true' : 'false'}
+            aria-expanded={menuOpen}
             aria-controls={idForLinkContainer}
-            aria-label={`${menuOpen ? 'close' : 'open'} anchor navigation menu`}
             data-testid={testIds.menuButton}
           >
             {menuOpen ? (
               <ChevronUpIcon size={16} className={styles['AnchorNav-menu-button-arrow']} fill="currentcolor" />
             ) : (
               <ChevronDownIcon size={16} className={styles['AnchorNav-menu-button-arrow']} fill="currentcolor" />
             )}
+            <span className="visually-hidden">Anchor navigation menu. Currently selected:</span>{' '}
```

Comment:
Nit: can this space be placed inside the visually hidden element instead?

Something like:

```suggestion
            <span className="visually-hidden">Anchor navigation menu. Currently selected: {' '}</span>
```